visidata/clipboard.py

```diff
@@ -106,6 +110,14 @@ def save(self, vs, filetype):
                 close_fds=True)
             p.communicate()
 
+def pasteFromClipboard(col_idx, row_idx, sheet):
+    text = pyperclip.paste()
```

Comment:
Hi @kovasap!  Thanks for submitting this PR.  We tried using pyperclip for the `syscopy` commands but there were issues (https://github.com/saulpw/visidata/pull/144).  Probably what we should do, is add a fourth element to `__clipboard_commands` for the "paste args" (so the third element becomes the "copy args".  Then a `_Clipboard.paste()` method would return the value on the system clipboard using the same kind of `subprocess.Popen` as in `copy()` (but in reverse).

Reply:
Ok cool.  I can throw that together.